cmd/medius/common/registry.go

```diff
@@ -24,6 +25,18 @@ var Registry = []Entry{
 		Artifact:   rhcos.New("4.9"),
 		UseForDocs: true,
 	},
+	{
+		Artifact:   prerelease.New("latest-4.9"),
```

Comment:
As discussed above, I think naming it `rhcosprerelease` would be a bit easier to understand here. (although it is idiomatic Go currently)